ovh/resource_okms_service_key_jwk.go

```diff
@@ -0,0 +1,178 @@
+package ovh
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	ovhtypes "github.com/ovh/terraform-provider-ovh/ovh/types"
+)
+
+var _ resource.ResourceWithConfigure = (*okmsServiceKeyJwkResource)(nil)
+
+func NewOkmsServiceKeyJwkResource() resource.Resource {
+	return &okmsServiceKeyJwkResource{}
+}
+
+type okmsServiceKeyJwkResource struct {
+	config *Config
+}
+
+func (r *okmsServiceKeyJwkResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_okms_service_key_jwk"
+}
+
+func (d *okmsServiceKeyJwkResource) Configure(_ context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+	if req.ProviderData == nil {
+		return
+	}
+
+	config, ok := req.ProviderData.(*Config)
+	if !ok {
+		resp.Diagnostics.AddError(
+			"Unexpected Resource Configure Type",
+			fmt.Sprintf("Expected *Config, got: %T. Please report this issue to the provider developers.", req.ProviderData),
+		)
+		return
+	}
+
+	d.config = config
+}
+
+func (d *okmsServiceKeyJwkResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = OkmsServiceKeyJwkResourceSchema(ctx)
+}
+
+func (r *okmsServiceKeyJwkResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	var data, responseData OkmsServiceKeyJwkResourceModel
+
+	// Read Terraform plan data into the model
+	resp.Diagnostics.Append(req.Plan.Get(ctx, &data)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	endpoint := "/v2/okms/resource/" + url.PathEscape(data.OkmsId.ValueString()) + "/serviceKey/"
+	if err := r.config.OVHClient.Post(endpoint, data.ToCreate(&resp.Diagnostics, ctx), &responseData); err != nil {
+		resp.Diagnostics.AddError(
+			fmt.Sprintf("Error calling Post %s", endpoint),
+			err.Error(),
+		)
+		return
+	}
+
+	data.MergeWith(&responseData)
+	// Save data into Terraform state
+	resp.Diagnostics.Append(resp.State.Set(ctx, &responseData)...)
```

Comment:
is there a reason to set `responseData` and not `data` in the state ?

Reply:
No good reason. Fixed.